plugins/basic/nexus-repository-httpbridge/src/main/java/org/sonatype/nexus/repository/httpbridge/internal/ViewServlet.java

```diff
@@ -193,7 +196,21 @@ Response describe(final Request request, final Response response, final Exceptio
     if (response != null) {
       DescriptionUtils.describeResponse(description, response);
     }
-    return toHtmlResponse(description);
+
+    DescribeType type = DescribeType.parse(flags);
+    log.trace("Describe type: {}", type);
+    switch (type) {
+      case HTML: {
+        String html = descriptionRenderer.renderHtml(description);
+        return HttpResponses.ok(new StringPayload(html, Charsets.UTF_8, "text/html"));
+      }
+      case JSON: {
```

Comment:
personally would have added test to verify content types and JSON payloads, and status codes, including for bad describe param